components/Typography/examples/TypographyMaterialScalableStylesViewController.m

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "TypographyMaterialScalableStylesViewController.h"
+
+#import "MaterialTypography.h"
+#import "MaterialTypographyScheme.h"
+
+@implementation TypographyMaterialScalableStyleViewController {
+  NSArray<NSString *> *_strings;
+  NSArray<NSString *> *_styleNames;
+  NSArray<UIFont *> *_styleFonts;
+  MDCTypographyScheme *_typography;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  self.tableView.separatorStyle = UITableViewCellSeparatorStyleNone;
+
+  self.tableView.rowHeight = UITableViewAutomaticDimension;
+  self.tableView.estimatedRowHeight = 50.0;
+
+  _strings = @[
+    @"Material Design Components", @"A quick brown fox jumped over the lazy dog.",
```

Comment:
Tip: consider using non-Latin characters to show rendering in other scripts. Suggestions: 
* A "CJK" language like Korean
* Cyrillic
* Hindi

Reply:
Good idea, added #6958 to track